fleet/variables.tf

```diff
@@ -55,3 +55,11 @@ variable "gateway_dns" {
 variable "user_data_file" {}
 
 variable "spot_user_data_file" {}
+
+variable "lb_stickiness_enabled" {
+  default = false
+}
+
+variable "lb_stickiness_cookie_duration" {
+  default = 86400
```

Comment:
I see this is one day.

Reply:
This is the default value set in Terraform and default for the module only. It should be provided in the recipe that uses the module.